llm_guard/input_scanners/anonymize_helpers/predefined_recognizers/zh/__init__.py

```diff
@@ -0,0 +1,13 @@
+from ..phone_recognizer import PhoneRecognizer
+from .crypto_recognizer import CryptoRecognizer
+from .custom_pattern_recognizer import CustomPatternRecognizer
+from .email_recognizer import EmailRecognizer
+from .ip_recognizer import IpRecognizer
+
+__all__ = [
```

Comment:
Is there a difference compared to the default recognizers or it's needed when you add a new language support?

Reply:
Because there is no concept of words (`\b` `\W` in regex) in Chinese, and perhaps some languages are also like this.

The default recognizers have much `\b` `\W` regex for words, they are not working for such languages.

So I extends the default recognizer, and adjust their patterns.

Reply:
For `it's needed when you add a new language support?`
---
Should confirm the language support `\b` `\W` regex. If support, not need. Otherwise, yes.

I think it's not a good solution, but I don't have a better idea.

Reply:
Makes sense now. I don't know Chinese, so was curious about challenges there.